client/post-editor/editor-notice/index.jsx

```diff
@@ -192,8 +195,30 @@ export class EditorNotice extends Component {
 		}
 	}
 
+	renderNoticeAction() {
+		const { isSitePreviewable, onViewClick, action, link } = this.props;
+
+		if ( isSitePreviewable ) {
+			return (
+				onViewClick && (
```

Comment:
If `onViewClick` is not passed in as a prop, should we fall back to the legacy preview functionality? At the very least could we move this conditional up to line 201?